content/concepts/consensus-network/amendments/known-amendments.md

```diff
@@ -77,9 +101,39 @@ The following is a comprehensive list of all known [amendments](amendments.html)
 | [SusPay][]                        | v0.31.0    | [Vetoed: Removed in v0.60.0](https://xrpl.org/blog/2017/ticksize-voting.html#upcoming-features "BADGE_RED") |
 | [Tickets][]                       | v0.30.1    | [Vetoed: Removed in v0.90.0](https://xrpl.org/blog/2018/rippled-0.90.0.html "BADGE_RED") |
 
-**Note:** In many cases, an incomplete version of the code for an amendment is present in previous versions of the software. The "Introduced" version in the table above is the first stable version. The value "TBD" indicates that the amendment is not yet considered stable.
+## Details about Known Amendments
+
 
-## CheckCashMakesTrustLine
+### AMM
+[AMM]: #amm
+
+| Amendment    | Automated Market Maker |
+|:-------------|:------------------------|
+| Amendment ID | 8CC0774A3BF66D1D22E76BBDA8E8A232E6B6313834301B3B23E8601196AE6455 |
+| Status       | In Development |
+| Default Vote (Latest stable release) | No |
+| Pre-amendment functionality retired? | No |
+
+Adds Automated Market Maker (AMM) functionality to the ledger in a way that is integrated with the existing decentralized exchange. Each pair of assets (tokens or XRP) can have up to one AMM in the ledger, which anyone can contribute liquidity to for a proportional share in the earnings and exchange risk. Each AMM instance has a special account to hold its assets and issue "LP Tokens" to liquidity providers in proportion to their deposits. Liquidity providers can vote on the AMM's trading fee based on their share of LP Tokens. Users can bid LP Tokens on the right to trade with a discounted trading fee for a limited period of time.
+
+Adds new transactions:
+
+- AMMBid - Bid on the AMM's auction slot, which offers discounted fees.
+- AMMCreate - Create a new AMM instance and provide initial funding.
+- AMMDelete - Remove an empty AMM instance from the ledger.
+- AMMDeposit - Add funds to an existing AMM and receive LP Tokens.
+- AMMWithdraw - Return LP Tokens to an AMM to remove funds.
+- AMMVote - Vote on the AMM's trading fee.
+
+Updates existing transactions with new functionality:
+
+- Payment and OfferCreate transactions that trade currency automatically use any combination of Offers and AMMs to achieve the best exchange rate.
+- Some transactions cannot be sent to an AMM's special account. (For example, the AMM cannot cash a check, so CheckCreate to an AMM is not allowed.)
+
+Adds a new type of ledger entry, `AMM`, and adds an `AMMID` field to the `AccountRoot` ledger entry type.
+
+Adds several new transaction result codes.
+### CheckCashMakesTrustLine
```

Comment:
Looks like applying my suggestion above caused this to be missing the lines separating it from the next one.

```suggestion
Adds several new transaction result codes.


### CheckCashMakesTrustLine
```